tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -17,7 +15,22 @@ const VIEWS = {
     Sleep: "Sleep"
 }
 const sideComponentWidth = 120
-const Calendar = () => {
+
+// The component that is used by each day in the calendar
+const DayComponent = ({ date, state, marking, navigation }) => {
```

Comment:
Curious, what does `state` and `marking` store? Otherwise, everything looks good!